### ISSUE TITLE
Recon fails on multiecho acquisition VB13C-SP2

### ISSUE THREAD

**Issue by ZhaolinChen** [CMRR-C2P/MB#190]
Hi, 

We would like to use multi-echo multiband EPI (R016a) on Skyra with VB13C-Sp2 software, but the recon fails to recon images.

Kind regards,
Zhaolin

**Comment by eauerbach** [comment]
Please collect a savelog after you see the error and send it to me.

**Comment by ZhaolinChen** [comment]
Hi Eddie,

Thanks, and please find the error saved from logviewer, and a screen shot of the error. Please let me know if you need more information.

Kind regards,
Zhaolin

**Comment by ZhaolinChen** [comment]
Hi Eddie,

Just want to follow up regarding this issue - have you had a chance to look into it? we are about to pilot a small cohort using the multiecho parameter so any suggestions would be much appreciated.

Kind regards,
Zhaolin

**Comment by handwerkerd** [comment]
I've been having a similar issue on a Prisma (VD-13 I think). I figured out a hack solution that might help localize the issue. If I run a multi-echo + MB sequence, it will acquire the data, but not reconstruct. If I then copy that sequence and open it to edit parameters, I get a box that says my TE's need to be changed by by a 0.01-0.03 milliseconds. I accept that change, save the new sequence and then it runs and reconstructs. This doesn't work by just saving and re-opening a sequence and it doesn't work by saving, copying, & opening the copy. I need to actually start the sequence running to have this error message pop up.

My current workflow is that I set the parameters I want, start the sequence. Stop it while it's still adjusting frequency. Copy it. Open the copy to adjust the TEs and then delete the original sequence. This has reliably worked for me, but it's a bit of a hack to get around whatever the underlying issue is.
